test/test_worker.py

```diff
@@ -20,9 +20,13 @@ def setUp(self):
 
     def test_proctitle(self):
         '''Make sure we can get / set the process title'''
-        before = Worker.title()
-        Worker.title('Foo')
-        self.assertNotEqual(before, Worker.title())
+        try:
+            import setproctitle
+            before = Worker.title()
+            Worker.title('Foo')
+            self.assertNotEqual(before, Worker.title())
+        except ImportError:
+            pass
```

Comment:
Or if you want to be explicit about it, you could skip the test from the except block.
```python
except ImportError:
    self.skipTest('setproctitle not available')
```